src/deep_merge.rs

```diff
@@ -56,16 +52,16 @@
 ///
 /// `self` is just replaced by `other`.
 pub trait DeepMerge {
-    /// Merge `other` into `self`.
-    fn merge_from(&mut self, other: Self);
+	/// Merge `other` into `self`.
```

Comment:
The code I wrote by hand uses tabs because I prefer tabs. But when generating code it's much more annoying to use tabs because that would require `\t` literals in the strings which are harder to read, so the generated code uses spaces. So for consistency with the generated code, other files in the `k8s-openapi` crate that are also hand-written (lib.rs etc) also use spaces.

Yeah, I know it's confusing. I should probably just convert everything to spaces later.

Reply:
a0c8d026617e6cad7390ece11fd758cefbd7ed30